common/changes/@itwin/one-click-lca-react/insights-update-oclca_2022-08-31-11-10.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/one-click-lca-react",
+      "comment": "Updated oclca widget to insights-client 0.3.0",
```

Don't use the shortname in comment.
```suggestion
      "comment": "Updated One Click LCA widget to use @itwin/insights-client version 0.3.0",
```

---

changed comment message